.gitignore

```diff
@@ -55,3 +55,6 @@ value
 
 # Folder created when using make
 cwltool_deps
+docs/_build/
+docs/autoapi/
```

Comment:
Annoyed by these files that appeared in my `git status` while testing this PR. Happy to drop this if it's better :+1: